sdk/reservations/Azure.ResourceManager.Reservations/CHANGELOG.md

```diff
@@ -1,15 +1,16 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0 (2023-02-15)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- Introduced overloads for some methods that contain quite a few parameters in their signatures.
```

Comment:
We use this comment `Introduced property bag for the methods with more than 5 parameters` for property bag, maybe we can align here?